tests/components/unifi/test_switch.py

```diff
@@ -373,36 +375,36 @@ async def test_option_block_clients(hass):
         options={CONF_BLOCK_CLIENT: [BLOCKED["mac"]]},
         clients_all_response=[BLOCKED, UNBLOCKED],
     )
-    assert len(hass.states.async_entity_ids("switch")) == 1
+    assert len(hass.states.async_entity_ids(SWITCH_DOMAIN)) == 1
 
     # Add a second switch
     hass.config_entries.async_update_entry(
         controller.config_entry,
         options={CONF_BLOCK_CLIENT: [BLOCKED["mac"], UNBLOCKED["mac"]]},
     )
     await hass.async_block_till_done()
-    assert len(hass.states.async_entity_ids("switch")) == 2
+    assert len(hass.states.async_entity_ids(SWITCH_DOMAIN)) == 2
 
     # Remove the second switch again
     hass.config_entries.async_update_entry(
         controller.config_entry, options={CONF_BLOCK_CLIENT: [BLOCKED["mac"]]},
     )
     await hass.async_block_till_done()
-    assert len(hass.states.async_entity_ids("switch")) == 1
+    assert len(hass.states.async_entity_ids(SWITCH_DOMAIN)) == 1
 
     # Enable one and remove another one
     hass.config_entries.async_update_entry(
         controller.config_entry, options={CONF_BLOCK_CLIENT: [UNBLOCKED["mac"]]},
     )
     await hass.async_block_till_done()
-    assert len(hass.states.async_entity_ids("switch")) == 1
+    assert len(hass.states.async_entity_ids(SWITCH_DOMAIN)) == 1
 
     # Remove one
     hass.config_entries.async_update_entry(
         controller.config_entry, options={CONF_BLOCK_CLIENT: []},
     )
     await hass.async_block_till_done()
-    assert len(hass.states.async_entity_ids("switch")) == 0
+    assert len(hass.states.async_entity_ids(SWITCH_DOMAIN)) == 0
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5403678)